java/smoketest/build.gradle

```diff
@@ -0,0 +1,24 @@
+
+apply plugin: 'java'
+apply plugin: 'application'
+
+mainClassName = 'SmokeTest'
+
+repositories {
+  flatDir {
+    dirs '../build/libs', '../cli/build/libs', '../core/build/libs'
+  }
+  mavenCentral()
+}
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+dependencies {
+  implementation "ai.whylabs:core:0.1.7-b2"
```

Comment:
How do we handle whylogs java version changes?

Reply:
The workflow needs something like the Python's bumpversion. I'm still inclined to keep the Python and Java version numbers the same so the packages don't collide when updating the other language.

Reply:
ok, let's follow up with the version bump outside this PR.